pulp_container/app/utils.py

```diff
@@ -122,3 +123,15 @@ def has_task_completed(dispatched_task):
             task.delete()
             raise Exception(str(error))
     raise Throttled()
+
+
+def get_media_type(content_data, response):
```

Comment:
can you rename it to `determine_media_type`? I think it reflects better the logic of the function